src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -442,7 +447,21 @@ export class DashboardAppController {
                 switchMap((newChildIds: string[]) =>
                   merge(
                     ...newChildIds.map((childId) =>
-                      dashboardContainer!.getChild(childId).getOutput$()
+                      dashboardContainer!
+                        .getChild(childId)
+                        .getOutput$()
+                        .pipe(
+                          catchError((error: Error) => {
+                            dashboardContainer?.replacePanel<Partial<ErrorEmbeddableInput>>({
+                              previousPanel: childId,
+                              newPanel: {
+                                type: ERROR_EMBEDDABLE_TYPE,
+                                explicitInput: { error },
+                              },
+                            });
+                            return of({});
```

Comment:
I think it is better to return `EMPTY` in this case? 
otherwise for a downstream it seems like all is fine just `output` is `{}`

Reply:
Updated this to use `EMPTY`, which is much cleaner, thanks for the suggestion!